.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,21 +19,17 @@ The changelogs will be integrated by the core maintainers after the merge.  See
 ### Submitter checklist
 
 - [ ] JIRA issue is well described
-- [ ] Link to JIRA ticket in description, if appropriate
+- [ ] Changelog entries provide a human-readable description of the change. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
```

Comment:
I don't think "human readable" captures what we want here.

If it adds a user visible feature or fixes a user visible bug, describe that effect for a user audience.
If it is about a developer/internal only behavior or feature, describe that effect for a developer audience.

How about this?

* Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change)

It doesn't need to be perfect, since we're going to use it as a guideline anyway (so tone and appropriate level of detail don't matter as much) – but it's important that the change is described for its audience.